src/data/accessibility-quiz.ts

```diff
@@ -1,4 +1,13 @@
 const accessibilityQuiz = [
+  {
+    Question: "Why is it important to add alternative text to images on web pages?",
```

Comment:
we already have a similar question on line 97

please choose another accessibility question to add instead

Reply:
Thanks for letting me know, I will change the question.👍